package.json

```diff
@@ -53,30 +53,34 @@
     ]
   },
   "dependencies": {
-    "@react-native-community/async-storage": "^1.5.0",
+    "@react-native-community/async-storage": "^1.6.2",
     "@react-navigation/core": "^3.5.1",
     "bugsnag-react-native": "2.23.2",
+    "lodash": "4.17.15",
     "prop-types": "^15.7.2",
-    "react": "^16.10.2",
-    "react-native": "0.61.2",
+    "react": "16.9.0",
```

Comment:
downgrade?

Reply:
Just the version RN wants - mainly did for peer dependency warning. Just bug fixes that I don't believe eeffect us